src/css/imports/controlbar.less

```diff
@@ -94,6 +94,9 @@
   display: none;
   margin: 0;
   padding: 0;
+  width: 32px;
```

Comment:
Use Command + / Command - (Menu > View > Zoom In/Out) to compare the icon to it's neighbors.